bravado_asyncio/future_adapter.py

```diff
@@ -13,6 +14,7 @@ class FutureAdapter(BaseFutureAdapter):
     a normal Python function, and we expect future to be from the concurrent.futures module."""
 
     timeout_errors = (concurrent.futures.TimeoutError,)
+    connection_errors = (aiohttp.client_exceptions.ClientError,)
```

Comment:
Shouldn't this be `aiohttp.ClientConnectionError`?

Reply:
`aiohttp.ClientConnectionError` extends `aiohttp.client_exceptions.ClientError` which is more generic

Reply:
Oh absolutely - I'm wondering if it's _too_ generic. :) But we can roll with it for now.

Reply:
By running the following script I've extracted the hierarchy graph of the mentioned exception classes
obtaining the following graph
![https://gist.githubusercontent.com/macisamuele/be2aea6df30142584b85fef5c382c533/raw/f43ecd37a9bf674445fe14c14ce40063f0d82334/hierarchy.svg](https://gist.githubusercontent.com/macisamuele/be2aea6df30142584b85fef5c382c533/raw/f43ecd37a9bf674445fe14c14ce40063f0d82334/hierarchy.svg?sanitize=true)
Script and data on https://gist.github.com/macisamuele/be2aea6df30142584b85fef5c382c533

Using `aiohttp.client_exceptions.ClientError` will be catching
* `aiohttp.client_exceptions.TooManyRedirects`:  _Client was redirected too many times._
* `aiohttp.client_exceptions.ClientPayloadError`:  _Response payload error._
* `aiohttp.client_exceptions.ClientHttpProxyError`:  _HTTP proxy error._
* `aiohttp.client_exceptions.ClientResponseError`:  _Connection error during reading response._
* `aiohttp.client_exceptions.WSServerHandshakeError`:  _websocket server handshake error._
* `aiohttp.client_exceptions.ContentTypeError`:  _ContentType found is not valid._
* `aiohttp.client_exceptions.InvalidURL`:  _Invalid URL. URL used for fetching is malformed, e.g. it doesn't contains host part._

Which I think sould not be considered connection errors.
I'm going to limit for now the exception set (as it will be easier to expand it if needed)